### ISSUE TITLE
Red axios config not working

### ISSUE THREAD

**Issue by lanceliamll** [developit/redaxios#27]
im currently using axios(CONFIGS_HERE) and suddenly content-type and headers doesnt work. 
tried to have a 
```  
headers: {
      'Content-Type': 'application/x-www-form-urlencoded',
    },
```
but it doesnt reflect on my API request. its still "application/json"

Please help.

**Comment by developit** [comment]
Are you using FormData? This seems like it might be the same issue as #28.

**Comment by developit** [comment]
Fixed in [version 0.3.0](https://github.com/developit/redaxios/releases/tag/0.3.0).

**Comment by lanceliamll** [comment]
Thank you sir!
